### DISCUSSION TITLE
v1.16.0 release

### DISCUSSION THREAD

**Discussion by andyzhangx** [kubernetes-sigs/blob-csi-driver#740] [Announcements]
### Features 🌈
 - support cross subscription with management API (#729)
 - upgrade blobfuse install version as v1.4.5 (#734)
 - add requireInfraEncryption parameter in storage class (#737)

### Bug Fixes 🐞
 - fix: CSI function should return internal error (#719)

## What's Changed
* doc: set master branch chart version as v0.0.0 by @andyzhangx in https://github.com/kubernetes-sigs/blob-csi-driver/pull/715
* doc: update chart index by @andyzhangx in https://github.com/kubernetes-sigs/blob-csi-driver/pull/717
* test: disable CVE-2022-32981 check by @andyzhangx in https://github.com/kubernetes-sigs/blob-csi-driver/pull/716
* fix: CSI function should return internal error by @andyzhangx in https://github.com/kubernetes-sigs/blob-csi-driver/pull/719
* add e2e test for sas token and key vault suppport by @cvvz in https://github.com/kubernetes-sigs/blob-csi-driver/pull/718
* chore: upgrade azure cloud provider lib by @andyzhangx in https://github.com/kubernetes-sigs/blob-csi-driver/pull/721
* e2e test for sas token by @cvvz in https://github.com/kubernetes-sigs/blob-csi-driver/pull/722
* Fixed typos in read-from-secret.md file by @bishal7679 in https://github.com/kubernetes-sigs/blob-csi-driver/pull/726
* test: fix TestLogGRPC ut failure by @umagnus in https://github.com/kubernetes-sigs/blob-csi-driver/pull/731
* feat: support cross subscription with management API by @cvvz in https://github.com/kubernetes-sigs/blob-csi-driver/pull/729
* test: add e2e test for provided credentials by @cvvz in https://github.com/kubernetes-sigs/blob-csi-driver/pull/730
* fix race in unit test and add some simple tests for utils by @avoltz in https://github.com/kubernetes-sigs/blob-csi-driver/pull/732
* cleanup: remove secrets list rbac by @andyzhangx in https://github.com/kubernetes-sigs/blob-csi-driver/pull/733
* feat: upgrade blobfuse install version as v1.4.5 by @andyzhangx in https://github.com/kubernetes-sigs/blob-csi-driver/pull/734
* fix: fixing codespell errors by @umagnus in https://github.com/kubernetes-sigs/blob-csi-driver/pull/735
* Improved Unit-Test Coverage for Issue #703 by @aarongalang in https://github.com/kubernetes-sigs/blob-csi-driver/pull/736
* feat: add requireInfraEncryption parameter in storage class by @andyzhangx in https://github.com/kubernetes-sigs/blob-csi-driver/pull/737
* doc: remove nconnect mount option by @andyzhangx in https://github.com/kubernetes-sigs/blob-csi-driver/pull/739

## New Contributors
* @bishal7679 made their first contribution in https://github.com/kubernetes-sigs/blob-csi-driver/pull/726
* @umagnus made their first contribution in https://github.com/kubernetes-sigs/blob-csi-driver/pull/731
* @aarongalang made their first contribution in https://github.com/kubernetes-sigs/blob-csi-driver/pull/736

**Full Changelog**: https://github.com/kubernetes-sigs/blob-csi-driver/compare/v1.15.0...v1.16.0

<hr /><em>This discussion was created from the release <a href='https://github.com/kubernetes-sigs/blob-csi-driver/releases/tag/v1.16.0'>v1.16.0 release</a>.</em>

**Comment by bishal7679** [comment]
Thank you @andyzhangx for mentioning me 💖
